### PR TITLE
Disable NNCF optimization for FP16 models

### PR DESCRIPTION
**Pull request by daniil-lyakhov**
FP16 model optimization by NNCF will be supported only in OTX2.0. Until then NNCF optimization for that models should be disabled.

### CODE REVIEW

**Comment by daniil-lyakhov** [comment]
CC: @goodsong81

File: external/model-preparation-algorithm/configs/instance-segmentation/efficientnetb2b_maskrcnn/template.yaml [hunk 1]
```diff
@@ -40,8 +40,9 @@ hyper_parameters:
       stat_requests_number:
         default_value: 2
     nncf_optimization:
+      visible_in_ui: false
       enable_quantization:
-        default_value: true
+        default_value: false
```
Comment (alexsu52) [inline external/model-preparation-algorithm/configs/instance-segmentation/efficientnetb2b_maskrcnn/template.yaml:43]: I'm not sure if default_value: false will disable NNCF task. Please, check it on the Geti side. As far as I remember, we removed nncf entrypoint: `mpa_tasks.apis.detection.DetectionNNCFTask` to disable the nncf task.

Comment (daniil-lyakhov) [reply external/model-preparation-algorithm/configs/instance-segmentation/efficientnetb2b_maskrcnn/template.yaml:43]: Done, I'll check results on a SC instance

**Comment by daniil-lyakhov** [comment]
Release branch PR: #1319

**Review by alexsu52 — APPROVED** [review]

**Review by goodsong81 — APPROVED** [review]

**Pull request merged**